Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/Game.java

```diff
@@ -0,0 +1,115 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+import java.util.ArrayList;
+
+public class Game {
+	protected 	Dictionary 	dictionary;		
+	protected 	Player 		player;		  	
+	protected 	char[] 		etalon;			
```

Comment:
Assuming you gain security, readability and maintainability. 

1. Security is the same in both implementations because I don't want for any classes other than t's children to have access to these fields. This can be done by having protected getters/setters or by just making the variables public. The access distribution would be the same. 
2. Maintainability. You'd just have extra functions, making  this file even larger. Plus most programmers are used to having public getters and setters. Having protected access functions will give them a headache.
3. Readability. This is arguable, but I think that doing `Dictionary = new FancyJavaDictionary ()` looks much better than `setDictionary(new FancyJavaDictionary())`;. This is indeed a matter of taste.